docs/DeveloperGuide.md

```diff
@@ -105,6 +105,8 @@ The `UI` component,
 * keeps a reference to the `Logic` component, because the `UI` relies on the `Logic` to execute commands.
 * depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
 
+The `PersonCard` part also has a reference to the `MainWindow` to provide improved user feedback via the main window after user interactions which do not execute a command using `Logic`.
+cc
```

Comment:
Need to remove this 'cc'

Reply:
Whoops, not sure how that got there.

Reply:
Removed in https://github.com/AY2122S1-CS2103T-F11-4/tp/pull/169/commits/72e7b39a9b3b4b6d8fa75d3c44d74ddfb3f72e6c.